baggers.js

```diff
@@ -1,5 +1,90 @@
 var data = {
   speakers: [
+    {
```

Comment:
Please move yourself at the end of the list.

Reply:
Done.